### ISSUE TITLE
Docker resource

### ISSUE THREAD

**Issue by gaow** [bpeng2000/SOS#83]
https://github.com/BioDocker/biodocker

This seems to be an ambitious project that we might take advantage of.

**Comment by bpeng2000** [comment]
This looks like a great resource. Did you find documentation about common interface? Assumptions about user name, input, output etc?

**Comment by gaow** [comment]
It seems there's only a number of docker files for users to download and use ... But is this also the case with ngseasy, and it is sufficient for SoS to get it work with the proper docker file?

**Comment by bpeng2000** [comment]
https://github.com/BioDocker/containers has a bunch.

The problem with providing Dockerfile only (not that they are not useful) is that the Dockerfile might expire when the needed resource becomes unavailable. I guess they have set up something like auto docker to build container automatically.

**Comment by gaow** [comment]
Yes, it does not look like it has an image now. But I feel this is an ambitious project which eventually may have all that we expect. Let's keep an eye on it :)

**Comment by bpeng2000** [comment]
github or dockerhub does have a feature that automatically build container from Dockerfile... I read it somewhere but do not remember the details. I guess you can try to download one and see if the container is available.

**Comment by tiagoantao** [comment]
Probably the fastest and safest solution (for users) is to build it at the docker hub and then pull it. So, no problems if a resource disappears. Also, while the pre-made container might be big in terms of network demand, it is less of a hassle than building locally.

Small containers can be built if using alpine Linux (current "official" distro for Docker). I think the footprint is around 10MB for a basic container!
